PilotDataPlatform/notification
# fix: Add full url in invitation html

## Summary

- Add full url in platform invitation html, which was missing under: `If you are having difficulty accessing the link please copy and paste the following into your browser:`

## JIRA Issues

N/A

## Type of Change


## Testing


## Test Directions

Email received for invitation to platform should have full url under `If you are having difficulty accessing the link please copy and paste the following into your browser:`
